### ISSUE TITLE
Preprocessing multi labeled data

### ISSUE THREAD

**Issue by efferre99** [keras-team/keras#11417]
I have to preprocess images that have multi label.
In particular I have to project a neural network that not only classifies two different shape, triangle or circle, but also get their dimensions: the side, the radius, the position etc.. It's seems with no sense, but for my application it has. Where can I find a similar example?
